jujugui/static/gui/src/app/components/scale-service/_scale-service.scss

```diff
@@ -0,0 +1,51 @@
+.scale-service {
+
+    input[type=text] {
+        margin: 20px;
+        height: 28px;
+    }
+
+    &__units {
+        position: relative;
+
+        input[type=text] {
```

Comment:
Can this element have `margin-bottom: 13px;` to bring the radio buttons in line with design.